pkg/apis/machine/v1alpha1/types.go

```diff
@@ -333,6 +349,9 @@ type MachineSetStatus struct {
 
 	// ObservedGeneration
 	ObservedGeneration int64 `json:"observedGeneration,inline"`
+
+	// FailedMachines has summary of machines on which lastOperation Failed
+	FailedMachines *[]MachineSummary `json:"failedMachines,inline"`
```

Comment:
why are the 'json' for failed machines diffferent between machineSet and machineDeployments?